freestar-plugin-react-native.podspec

```diff
@@ -15,6 +15,6 @@ Pod::Spec.new do |s|
   s.source       = { :git => "https://gitlab.com/freestarcapital/freestar-plugin-react-native.git", :tag => "v#{s.version}" }
   s.source_files  = "ios/**/*.{h,m}"
 
-  s.dependency 'FreestarAds', '~> 4.0'
+  s.dependency 'FreestarAds', '~> 5.15.0'
```

Comment:
Updated to the latest Core Release https://github.com/freestarcapital/ios-core-sdk/releases/tag/5.15.0